app/routers/weight.py

```diff
@@ -0,0 +1,67 @@
+from typing import Union
+
+from fastapi import APIRouter, Depends, Request
+from sqlalchemy.orm import Session
+from starlette.responses import RedirectResponse
+
+from app.database.models import User
+from app.dependencies import get_db
+from app.dependencies import templates
+
+
+router = APIRouter(tags=["weight"],)
+
+
+@router.get("/weight")
+async def get_weight(
+        request: Request,
+        session: Session = Depends(get_db),
+        target: Union[float, None] = None,
```

Comment:
Can't I state a weight only with int?
For example 80?

Reply:
Yes, you can.
The float allow you enter also  79.5